### PR TITLE
services/horizon: Rename column names to be consistent

### LINKED ISSUE #1858: Make column names consistent
In ledger entries tables we use several different names for the same column types (ex. `account`, `accountid`, `account_id`). We should make it consistent.

### PR DESCRIPTION
**Pull request by bartekn**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [x] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [x] This PR adds tests for the most critical parts of the new functionality or fixes.
* [x] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [x] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [x] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

This commit is changing column names in tables used by `expingest` package to be consistent.

`22_trust_lines.sql` and `24_accounts.sql` have been changed directly because these migration files have not been released yet.

Close #1858.

### CODE REVIEW

File: services/horizon/internal/db2/history/account_data.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ func (q *Q) GetAccountDataByKeys(keys []xdr.LedgerKeyData) ([]Data, error) {
 		}
 		lkeys = append(lkeys, lkey)
 	}
-	sql := selectAccountData.Where(map[string]interface{}{"accounts_data.lkey": lkeys})
+	sql := selectAccountData.Where(map[string]interface{}{"accounts_data.ledger_key": lkeys})
```
Comment (abuiles) [inline services/horizon/internal/db2/history/account_data.go]: thanks for this! `ledger_key` is way better than `lkey`

File: services/horizon/internal/db2/history/main.go [hunk 2]
```diff
@@ -524,14 +524,14 @@ type TransactionsQ struct {
...
-	AssetCode          string        `db:"assetcode"`
+	AccountID          string        `db:"account_id"`
+	AssetType          xdr.AssetType `db:"asset_type"`
+	AssetIssuer        string        `db:"asset_issuer"`
+	AssetCode          string        `db:"asset_code"`
 	Balance            int64         `db:"balance"`
-	Limit              int64         `db:"tlimit"`
-	BuyingLiabilities  int64         `db:"buyingliabilities"`
-	SellingLiabilities int64         `db:"sellingliabilities"`
+	Limit              int64         `db:"trust_line_limit"`
```
Comment (abuiles) [inline services/horizon/internal/db2/history/main.go]: I always have issues when writing trustline, do we say `trust_line` or `trustline`? It seems like we use  both indifferently all around

File: services/horizon/internal/scripts/build_test_scenarios.bash [hunk 3]
```diff
@@ -3,8 +3,8 @@ set -e
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 GOTOP="$( cd "$DIR/../../../../../../../.." && pwd )"
-PACKAGES=$(find $GOTOP/src/github.com/stellar/go/services/horizon/internal/test/scenarios -iname '*.rb' -not -name '_common_accounts.rb')
-#PACKAGES=$(find $GOTOP/src/github.com/stellar/go/services/horizon/internal/test/scenarios -iname 'failed_transactions.rb')
+#PACKAGES=$(find $GOTOP/src/github.com/stellar/go/services/horizon/internal/test/scenarios -iname '*.rb' -not -name '_common_accounts.rb')
+PACKAGES=$(find $GOTOP/src/github.com/stellar/go/services/horizon/internal/test/scenarios -iname 'base.rb')
```
Comment (abuiles) [inline services/horizon/internal/scripts/build_test_scenarios.bash]: what is changing here?

File: services/horizon/internal/db2/history/main.go (hunk 2 above)
Comment (bartekn) [reply services/horizon/internal/db2/history/main.go]: It's `trust_line` or `trustLine` in stellar-core let's stick to this.

**Pull request merged**